department-of-veterans-affairs/caseflow
# JHoang/APPEALS-34680

Resolves [34680](https://jira.devops.va.gov/browse/APPEALS-34680?workflowName=APPEAL+Story+Workflow&stepId=20)

# Description
Direct user to step 3 of intake appeal process with step 1 and 2 completed.

## Acceptance Criteria

## Testing Plan
1. Login as MEAWADBVAJ.
2. In the address bar go to `/queue/correspondence`
3. Click on any correspondence with 10182 package document type (500000001 - 500000010)
4. Click on the Intake appeal button
4. Observe that user is directed to Step 3 of intake appeal process
5. Confirm that Step 1 and Step 2 is completed by checking the veteran name and file number at the top of the page


# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
